java/src/apps/AppsBundle_es.properties

```diff
@@ -1,4 +1,5 @@
-# AppsBundle.properties
+# java/src//apps/AppsBundle_es.properties
+# java/src/apps/AppsBundle.properties
```

Comment:
This doesn't look right. Shouldn't one of these lines be removed?

Reply:
That 2nd line was copied from the default file. I suppose it would be possible for a script to look for that comment and remove it, but it's not trivial:  There are a lot of possible forms. That might be a good future improvement.